### DISCUSSION TITLE
How to get common pvc for all nodes?

### DISCUSSION THREAD

**Discussion by Yaswani341** [Opster/opensearch-k8s-operator#819] [Q&A]
Im trying fs snapshot backup and i gave path.repo: "/mnt/snapshots". How can we add pvc so that it is mounted in all master and data nodes?

Im getting this error:
`"error" : {
    "root_cause" : [
      {
        "type" : "repository_verification_exception",
        "reason" : "[fs_repo_1] [[pQjlvAazRdGWTQ9_KAbMUQ, 'RemoteTransportException[[my-cluster-nodes-1][192.168.195.212:9300][internal:admin/repository/verify]]; nested: RepositoryVerificationException[[fs_repo_1] a file written by master to the store [/mnt/snapshots] cannot be accessed on the node [{my-cluster-nodes-1}{pQjlvAazRdGWTQ9_KAbMUQ}{8u8ynEQXRCGxpqRrRLwxrw}{my-cluster-nodes-1}{192.168.195.212:9300}{d}{shard_indexing_pressure_enabled=true}]. This might indicate that the store [/mnt/snapshots] is not shared between this node and the master node or that permissions on the store don't allow reading files written by the master node]; nested: NoSuchFileException[/mnt/snapshots/tests-dJ-a11kJTS64ipTk3cTiTw/master.dat];'], [xScW2b8hR22KnCNYFro2eA, 'RemoteTransportException[[my-cluster-nodes-0][192.168.144.164:9300][internal:admin/repository/verify]]; nested: RepositoryVerificationException[[fs_repo_1] a file written by master to the store [/mnt/snapshots] cannot be accessed on the node [{my-cluster-nodes-0}{xScW2b8hR22KnCNYFro2eA}{8fn8R87NT1qrATX6kXifYw}{my-cluster-nodes-0}{192.168.144.164:9300}{d}{shard_indexing_pressure_enabled=true}]. This might indicate that the store [/mnt/snapshots] is not shared between this node and the master node or that permissions on the store don't allow reading files written by the master node]; nested: NoSuchFileException[/mnt/snapshots/tests-dJ-a11kJTS64ipTk3cTiTw/master.dat];'], [-UursipGQhq-f4VmAdZShw, 'RemoteTransportException[[my-cluster-masters-1][192.168.161.67:9300][internal:admin/repository/verify]]; nested: RepositoryVerificationException[[fs_repo_1] a file written by master to the store [/mnt/snapshots] cannot be accessed on the node [{my-cluster-masters-1}{-UursipGQhq-f4VmAdZShw}{VFARD2T9Qc26t5dMX9b6Wg}{my-cluster-masters-1}{192.168.161.67:9300}{dm}{shard_indexing_pressure_enabled=true}]. This might indicate that the store [/mnt/snapshots] is not shared between this node and the master node or that permissions on the store don't allow reading files written by the master node]; nested: NoSuchFileException[/mnt/snapshots/tests-dJ-a11kJTS64ipTk3cTiTw/master.dat];'], [YFf5_ti9QMulGhG2i3tZPw, 'RemoteTransportException[[my-cluster-masters-2][192.168.144.154:9300][internal:admin/repository/verify]]; nested: RepositoryVerificationException[[fs_repo_1] a file written by master to the store [/mnt/snapshots] cannot be accessed on the node [{my-cluster-masters-2}{YFf5_ti9QMulGhG2i3tZPw}{fk4_0REWRZ2iq71gw9ueIA}{my-cluster-masters-2}{192.168.144.154:9300}{dm}{shard_indexing_pressure_enabled=true}]. This might indicate that the store [/mnt/snapshots] is not shared between this node and the master node or that permissions on the store don't allow reading files written by the master node]; nested: NoSuchFileException[/mnt/snapshots/tests-dJ-a11kJTS64ipTk3cTiTw/master.dat];']]"
      }
`

**Comment by mame-leadec** [comment]
@Yaswani341 i face the same issue. Did you found a solution?

**Comment by swoehrl-mw** [comment]
Closing as this was discussed in the linked issue.
